### DISCUSSION TITLE
Sponsor your work

### DISCUSSION THREAD

**Discussion by mmadson** [AnWeber/httpyac#685] [General]
Howdy @AnWeber 

Popping in again just to say thank you for all the work you've done on this project. I see you updating it on such a frequent basis and it's such a valuable tool. Even the relatively niche Junit test output support we added together a while back -- you've adopted it wholeheartedly and have continued to support it with additional updates and bugfixes. Such a rare trait in an OSS maintainer and it's just such a joy to use httpyac.

I figure if there was ever a maintainer / OSS project worthy of supporting financially it's you and httpyac. I haven't done any research to know what a good platform is for collecting sponshorships/donations but I believe github has a built in capability:

- https://github.com/sponsors

Regardless, thanks again for all you do!

Have a great weekend.

**Comment by AnWeber** [comment]
Hi @mmadson, 

nice to hear from you again. Nice of you to ask if you can sponsor me. I'm just not sure I want to go down that route. At the moment the whole project is non-binding. It gives me pleasure and my little escape to just start programming. But I'm under no obligation to anyone. In other words, I can simply prioritize time with my children or my wife at any time. But I'm still thinking about setting up a sponsorship..
There were still a few problems with Junit, but not your code. We just started from the wrong XML schema and then I had an error that had been hidden in my logic for some time. 

Best regards
